# Lecture 16. Chemistry of replication and transcription: polymerases

- Readings
  - pp. 119-123
  - pp. 827-844
  - pp. 859-880
  - pp. 925-931
  - pp. 941-947

## pp. 119-123

### 4.5<SPACE/> Gene expression the transformation of DNA information into functional molecules
- **80% of RNA in body is rRNA**
  - 23S rRNA: 1200 kDa (3700 nt)
  - 16S rRNA: 550 kDa (1700 nt)
  - 5S rRNA: 36 kDa (120 nt)
- 15% is **tRNA: 4S, 75 nt long**
- in eukaryotes, mRNA has **structural features** (e.g. **stem-loop structures**) that regulate efficiency of translation and lifetime of mRNA

#### All cellular RNA is synthesized by RNA polymerases
- mRNA is a exact copy of the 5' to 3' strand (sense, 5' to 3', coding strand)

#### Transcription begins near promoter sites and ends at terminator sites
- **promoters**. bind RNA polymerases

![Promoter sites in prokaryotes and eukaryotes.](./promoters.png)

- eukaryotes also have **enhancers** that mediate transcription
- in prokaryotes, transcription of a **stop sequence** spontaneously dissociates RNA from RNA polymerase
- RNA synthesis can also be terminated by a protein called **rho (Rho factor)**
- in eukaryotes, mRNA modifications
  - mRNA cap: 5'-5' triphosphate linkage
  - poly(A) tail

![Structure of mRNA cap.](./cap.png)

#### Transfer RNAs are the adaptor molecules in protein synthesis
- tRNAs have an AA-attachment site (AAAS) and template recognition site (TRS)
- **AAAS**
  - amino acid COOH group is esterfied to **2' or 3' OH group** (3' end of tRNA) on **adenosine (A)** at end of tRNA
    - called an **aminoacyl-tRNA**
  - attachment is catalyzed by **aminoacyl-tRNA synthetase**
- **template recognition site (TRS)**
  - 1 **anticodon**, 3' to 5'

![tRNA attachment at the 3' end (CCA-3').](./trna-attachment.png)

## pp. 827-844

### 28<SPACE/>DNA replication, repair, and recombination
- error rate <1 bp every $6 \times 10^9$ bp
  - multiple layers:
    - synthesis: 1 bp every $10^3$ to $10^4$
    - proofreading: 1 bp every $10^6$ to $10^7$
    - post-replication mismatch repair: 1 bp every $10^9$ to $10^{10}$
- _Deinococcus radiodurans_ has between 4 to 10 copies of its genome
  - can withstand high doses of radiation
  - can withstand dessication

### 28.1<SPACE/>DNA replication proceeds by the polymerization of dNTPs along a template
- difficulties
  1. replication needs to account for 5' to 3' direction on both strands
  2. strands must separate for replication
  3. DNA is a double helix, so strands needs to be unwound

#### DNA polymerases require a template and a primer
- DNApols catalyze **NA by 3'-OH group of polynucleotide chain** to the $\alpha$-phosphoryl group of the NTP to be added
  - this means a strand/primer must already exist

#### All DNA polymerases have structural features in common
- all DNA pols have **Klenow fragment**
  - half is **polymerase** domain
  - other half is **3' exonuclease** domain for proofreading
  - fingers, thumb grip DNA; **palm has active site**

![The Klenow fragment: it looks like a right hand, with an exonuclease unit.](./klenow-fragment.png)

#### Two bound metal ions participate in the polymerase reaction
- **all enzymes with NTP substrates** require metal ions for activity
  - one metal binds both primer and dNTP (left)
    - aligns primer's 3' OH to attack $\alpha$ phosphate of dNTP
  - one metal binds only dNTP (right)
    - stabilizes leaving pyrophosphate
  - both stabilize transition state

![DNA polymerase mechanism.](./dnapol-mxn.png)

#### The specificity of replication is dictated by complementarity of shape between bases
- hydrogen bonding helps, but **shape complementarity** is the main driver of specificity
- why is shape complementarity so important?
  - enzymes check the minor groove, which selects for **proper spacing**
  - finger clamps down; if not Watson-Crick pair, then won't form a tight conformation

![Minor groove hydrogen bonds.](./minor-groove.png)

![Conformational changes requiring shape selectivity.](./shape-selectivity.png)

#### An RNA primer synthesized by primase enables DNA synthesis to begin
- **primer**. a nucleic acid with a free 3' end that forms a double helix with the template

#### One strand of DNA is made continuously, whereas the other strand is synthesized in fragments
- all known DNA polymerases synthesize in the 5' $\to$ 3' direction
- the **leading strand** is extending from 5' $\to$ 3', but the **lagging strand** moves from 3' $\to$ 5'

![Okazaki fragments.](./okazaki.png)

#### DNA ligase joins ends of DNA in duplex regions
- DNA ligase cannot link ssDNA or circularize ssDNA
- circular dsDNA exists
- powered by ATP in eukaryotes and archaea; NAD+ in bacteria
- _E. coli_ DNA ligase ligates when overhangs are present
- **T4 bacteriophage** works for **blunt-ended dsDNA fragments**

#### The separation of DNA strands requires specific helicases and ATP hydrolysis
- **helicases require ATP**
- helicase structure
  - hexamers
  - core has P-loop NTPase
  - core also has DNA binding lop
- asymmetric; allows for cycling

![Structure of helicase.](./helicase.png)
![Helicase mechanism for unwinding DNA.](./helicase-mxn.png)

### 28.2<SPACE/>DNA unwinding and supercoiling are controlled by topoisomerases
- **right-handed superhelix** is called **negative superhelix**
- a supercoiled DNA molecule is more compact than a relaxed one of the same length
- supercoiled DNA moves faster in electrophoresis and centrifugation

#### The linking number of DNA, a topological property, determines the degree of supercoiling
- $L$. **linking number**. number of times a strand of DNA winds in the right-handed direction around the helix axis
- $T$. **twisting number**. measures the winding of strands around each other
- $W$. **writhing number**. measures coiling of the helix **axis** (i.e. supercoiling)
  - for super coils, right handed is negative
- **topoisomers**. molecules differing only in linking number
- $L = T + W$
  - $W = 0.7 \Delta L$ 

#### Topoisomerases prepare the double helix for unwinding

#### Type I topoisomerases (TOP1) relax supercoiled structures

#### Type II topoisomerases (TOP2) can introduce negative supercoils through coupling to ATP hydrolysis

### 28.3<SPACE/>DNA replication is highly coordinated

#### DNA replication requires highly processive polymerases

#### The leading and lagging strands are synthesized in a coordinated fashion

#### DNA replication in _E. coli_ begins at a unique site

#### DNA synthesis in eukaryotes is initiated at multiple sites

#### Telomeres are unique structures at the ends of linear chromosomes

#### Telomeres are replicated by telomerase, a specialized polymerase that carries its own RNA template

## pp. 859-886

### 29<SPACE/>RNA synthesis and processing

#### RNA synthesis comprises three stages: initiation, elongation, and termination

### 29.1<SPACE/>RNA Polymerases catalyze transcription

#### RNA chains are formed _de novo_ and grow in the 5' to 3' direction

#### RNA polymerases backtrack and correct errors

#### RNA polymerase binds to promoter sites on the DNA template to initiate transcription

#### RNA polymerases must unwind the template double helix for transcription to take place

#### Elongation takes place at transcription bubbles that move along the DNA template

#### Sequences within the newly transcribed RNA signal termination

#### Some mRNAs directly sense metabolite concentrations

#### The _rho_ protein helps to terminate the transcription of some genes

#### Precursors of tRNA nd rRNA are cleaved and chemically modified after transcription in prokaryotes

### 29.2<SPACE/>Transcription in eukaryotes is highly regulated

#### Three types of RNA polymerase synthesize RNA in eukaryotic cells

#### Three common elements can be found in the RNA polymerase II promoter region

#### The TFIID protein complex initiates the assembly of the active transcription complex

#### Multiple transcription factors interact with eukaryotic promoters

#### Enhancer sequences can stimulate transcription at start sites thousands of bases away

### 29.3<SPACE/>The transcription products of eukaryotic polymerases are processed

#### RNApol1 produces three ribosomal RNAs

#### RNApol3 produces transfer RNA

#### The product of RNApol2, the pre-mRNA transcript, acquires a 5' cap and 3' poly(A) tail

#### Small regulatory RNAs are cleaved from larger precursors

#### RNA editing changes the proteins encoded by mRNA

#### Sequences at the ends of introns specify splice sites in mRNA precursors

#### Splicing consists of two sequential transesterification reactions

#### Small nuclear RNAs in spliceosomes catalyze the splicing of mRNA precursors

#### Transcription and processing of mRNA are coupled

#### Mutations that affect pre-mRNA splicing cause disease

#### Most human pre-mRNAs can be spliced in alternative ways to yield different proteins

## pp. 925-931

### 31<SPACE/>The control of gene expression in prokaryotes

### 31.1<SPACE/>Many DNA-binding proteins recognize specific DNA sequences

### 31.2<SPACE/>Prokaryotic DNA-binding proteins bind specifically to regulatory sites in operons

## pp. 941-947

### 32<SPACE/>The control of gene expression in eukaryotes

### 32.1<SPACE/>Eukaryotic DNA is organized into chromatin

#### The helix turn helix motif is common to many prokaryotic DNA-binding proteins

### 32.2<SPACE/>Transcription factors bind DNA and regulate transcription initiation

#### An operon consists of regulatory elements and protein-encoding genes

#### The _lac_ repressor protein in the absence of lactose binds to the operator and blocks transcription

#### Ligand binding can induce structural changes in regulatory proteins

#### The operon is a common regulatory unit in prokaryotes

#### Transcription can be stimulated by proteins that contact RNA polymerase

---

## Lecture notes
 
### Key points
1. Base stacking, hypochromicity, DNA melting
2. Discovery of DNA Pol I and mechanism
3. Pol I activities: 5' to 3'polymerase, 3' to 5' exonuclease, 5' to 3' exonuclease
4. Modern view of Pol III and DNA replicatino
5. RNA structure, RNA vs. DNA polymerases

### DNA structure
- **stabilized by $\pi$-$\pi$ stacking**