# Lecture 4. Protein Purification 2, Protein Structure 1

- Readings: _pp. 35-49, 59-62, 85-92_

## Readings

### pp. 35-49

#### 2.3<SPACE />Secondary structure: polypeptide chains can fold into regular structures such as the alpha helix, beta sheet, turns, and loops
- beta turn
- omega loop
- **the alpha helix is a coiled structure stabilized by intrachain hydrogen bonds**
  - **4 residues away**
    - 3.6 residues per turn of helix
    - rise per residue: $\pu{1.5 \ang}$
      - pitch = $\pu{3.6 \times 1.5 \ang = 5.4 \ang}$
  - **side chains point outward**
  - destabilize $\alpha$-helix:
    - branching at $\beta$-carbon
      - valine, threonine, isoleucine
    - hydrogen-bond donors/acceptors that compete for main-chain NH and CO groups
      - serine
      - aspartate
      - asparagine
    - no NH group, ring structure
      - **proline** (only a $\alpha$-helix)
- **beta sheets are stabilized by hydrogen bonding between polypeptide strands**
- **beta turn**
  - 4 residues
  - hydrogen bond between CO of $i$ and NH of $i + 3$
- **fibrous proteins provide structural support for cells and tissues**
  - **keratin**
    - repeating seven amino acid sequence called **heptad repeat**
    - every seventh residue is **leucine**
  - **collagen**
    - glycine, proline, hydroxyproline
    - no hydrogen bonds
    - stabilized by steric repulsion of pyrrolidine rings of proline and hydroxyproline
    - superhelical cable of 3 strands is stabilized by hydrogen bonds
    - only glycine can fit inside the cable; the prolines face outwards from the cable

#### 2.4<SPACE />Tertiary structure: Water-soluble proteins fold into compact structures with nonpolar cores
- in aqueous solution, protein folding is driven by the tendency of hydrophobic residues to be excluded from water
- most NH and CO grups are paired in hydrogen bonding if they are to be located in the interior of a protein, away from water
- porins: exceptions that prove the rule
- **motifs**. combinations of secondary structure present in many proteins
  - **helix-turn-helix**, common in DNA-binding proteins
- **domain**. a region of a polypeptide chain that folds into a compact globular unit and connects to other parts of the chain via a flexible segment of polypeptides.

#### 2.5<SPACE />Quaternary structure: polypeptide chains can assemble into multisubunit structures
- refers to the spatial arrangement of subunits

### pp. 59-62

#### Summary
- important properties of peptide bonds
  - resistant to hydrolysis (kinetically)
  - planar
  - contains a hydrogen-bond donor and acceptor
  - uncharged, allowing for close packing
- visualizing proteins
  - **space filling models**. useful for visualizing conformational changes
  - **backbone models**. shows overall coruse of polypeptide chain
  - **ribbon diagram**. shows secondary structure
    - cylinders for membrane spanning $\alpha$-helices

### pp. 85-92

#### 3.3<SPACE />Mass spectrometry is a powerful technique for the identification of peptides and proteins
- **mass spec** lots us measure the atomic composition of an _unknown_ molecule
- three components
  - ion source
  - mass analyzer
  - detector
- how it works
  1. convert analyte molecules into gaseous, charged forms (**gas-phase ions**)
  2. apply electrostatic potentials to measure $m/z$ (mass-to-charge) ratio
- ion sources perform step 1
  - until recently, proteins could not be ionized efficiently because of **high molecular weight and low volatility**
  - **MALDI** (matrix-assisted laser desorption/ionization)
    - analyte evaporated in the presence of the **matrix** (a volatile, aromatic compound)
    - laser vaporizes the matrix
  - **ESI** (electrospray ionization)
    - droplets are passed through charged nozzle
    - low pressure to vaporize
- **Peptides can be sequenced by mass spec**
  - traditionally, peptides were sequenced with **Edman degradation**
    - N-terminal amino acid labeled with **phenyl isothiocyanate**
    - cleavage yields **phenylthiohydantoin (PTH)**-amino acid derivative
  - **tandem mass spec**
    - break **precursor ions** with **helium/argon gas** and measure new fragments (**product ions**)

![Tandem mass spectrometry for protein sequencing.](./tandem-mass-spec.png)

- **proteins can be specifically cleaved into small peptides to facilitate analysis**

![Specific cleavage sites.](./cleavage.png)

- SDS-PAGE can be used to determine the number of distinct polypeptide chains
- **genomic and proteomic methods are complementary**
  - DNA sequence cannot tell us about post-translational modifications/disulfide bonds
- **the amino acid sequence of a protein provides valuable information**
  1. compare with other known sequences for similarities
    - protein families can help us infer structure and function
  2. compare same protein in different species for evolutionary pathways
  3. search for internal repeats
    - reveals history of protein itself
  4. many proteins have signal sequences, which tell us where it localizes
  5. sequence data provides a basis for preparing antibodies
  6. amino acid sequences enable DNA probes for the encoding genes
- **individual proteins can be identified by mass spec**
  - protein cleavage + chromatographic separation + mass spec = **peptide mass fingerprinting**

## Questions
- Why do we need tandem mass spec for protein sequencing? (pp. 87)
- How do you make a DNA probe from only the amino acid sequence? (pp. 91)