# Module 1: Genotyping &ndash; Gels, PCR, Sequencing

## Readings

### [Assessment of Nucleic Acid Purity](https://tools.thermofisher.com/content/sfs/brochures/TN52646-E-0215M-NucleicAcid.pdf)
- for proteins, $\lambda_{\max} = \pu{280 nm}$
- for nucleic acids (DNA/RNA), $\lambda_{\max} = \pu{260 nm}$

- $A_{260}/A_{280} = \begin{cases}1.8 & \text{pure DNA}\\ 2.0&\text{pure RNA}\end{cases}$
- $A_{260}/A_{230} \in [2.0, 2.2]$; $A_{230}$ should be the trough.

- [**TE buffer (TE)**](https://en.wikipedia.org/wiki/TE_buffer).
  - TE is derived from **Tris**, a common pH buffer, and **EDTA**, a molecule that chelates cations like $\ce{Mg^2+}$.
  - Used to solubilize DNA/RNA while protecting it from degradation.

### PCR
- _Taq_ vs. _Pfu_ DNA polymerase
  - _Pfu_ has higher fidelity when copying DNA
  - both heat resistant

## Design a genome edit
- Design a genome edit and describe the rationale
  - Kind of edit
    - Knockout
    - Introduction of STOP codon
    - Insertion
    - Point mutation
  - Which gene?
  - Which organism?
- Attach CRISPR gRNA sequence or TALEN
- Genome editing: studying **causality**
  - e.g. Live pigs without PERV activity
- **Ideal genome editor properties**
  1. efficient and specific
      - on-target
      - minimize off-target
  2. programmability
      - ability to target any DNA sequence
  3. ability to introduce any type of mutation
      - insertion
      - deletion
      - point mutation
  4. practical
      - easy to design and construct

### Ways to modify DNA
- Homology directed repair (HDR)
  - has part of **left cut, right cut, and new sequence**
- Non-homologous end joining (NHEJ)

#### TALENs
- sequence binding via an array of TAL effector domains
  - 1 TAL for 1 nucleotide
- limitations
  - target sequence has to start with T

#### CRISPR
- **PAM**. (`NGG`) protospacer adjacent motif
  - newer editors can get rid of this requirement
- Cas9 causes a ds break

### Base editing
- around half of known pathogenic genetic variants are due to **SNV** (point mutations)

## Lab 2

### Hours
- 2:30PM-7PM
- 12-5PM, 1-6PM

### Gel
- 1% agarose
- company will do the Sanger sequencing
  - cheap, a couple of dollars
- Sanger sequencing and restriction digest are two ways of answering the **same question**
- TBE buffer is less prone to overheating and is better for longer electrophoresis runs
  - Tris-Borate-EDTA (TBE buffer)
  - Tris-Acetate-EDTA (TAE buffer)

### Molecular genetic basis of the histo-blood group ABO system
- Three kinds of carbohydrate antigens: **A, B, H**.
- Convert H antigen to A, B, or AB antigen
- **alloimmunity**. an immune response to _nonself antigens_ from _members of the same species_.
- **O blood group**
  - single base deletion; leads to inactive protein
- 9q34.1 to 9q
  - exon 6 contains the deletion that is found in most O alleles and results in a loss of enzymatic activity
  - exon 7 contains most of the coding sequence
- Genetic basis
  - ABO gene is on Chr9
  - A group subtypes: $\mathrm{A^1, A^2}$
  - 7 exons in ABO gene, exons 6 and 7 encode >75% of protein-coding sequence
  - exon 6 pattern: allows us to determine $\mathrm{O^1}$ allele
    - converts BstEII site to KpnI
  - exon 7: can distinguish among $\mathrm{A^1, A^2, B, O^2}$ alleles.
    - digest with HpaII
  - exon 6: **nt G261**
  - exon 7: **nt 467, 703, 1096**
- A-type people: **$\alpha 1\to 3$ N-acetylgalactosaminyltransferase (A transferase)**
- B-type people: **$\alpha 1\to 3$ galactosyltransferase (B transferase)**
- mutations
  - residues **176, 235, 266, 268**
  - deletion of **base 258**
- **Buffy coat**. the fraction of an anticoagulated blood sample that contains most of the white blood cells and platelets following centrifugation
- What is the **FY31/FY43 primer**?
  - **Fy glycoprotein**

#### Gel interpretation
- Exon 6
  - We digest the PCR product
    - in one lane, in BstE II only
    - in another lane, in Kpn I only
  - If we see cuts in both lanes, then heterozygous
  - If only cuts in BstEII lane: any of $\mathrm{A^1, A^2, B, O^2}$
  - If only cuts in KpnI lane: $\mathrm{O^1}$
- Exon 7
  - **SNPs that are in linkage disequilibrium with the SNPs at nucleotides 796 and 803 are assayed** (because inexpensive restriction enzymes for these SNPs exist, and the SNPs are genetically linked to the causative SNPs)
    - the SNPs assayed are **467, 703, 1096**