# Lecture 17. Chemistry of transcription: splicing, mRNA processing

- Readings
  - pp. 876-890

## pp. 876-890

### 29.3<SPACE/>The transcription products of eukaryotic polymerases are processed

#### RNA polymerase I produces three ribosomal RNAs

#### RNA polymerase III produces transfer RNA

#### The product of RNA polymerase II, the pre-mRNA transcript, acquires a 5' cap and a 3' poly(A) tail

#### Small regulatory RNAs are cleaved from larger precursors

#### RNA editing changes the proteins encoded by mRNA

#### Sequences at the ends of introns specify splice sites in mRNA precursors

#### Splicing consists of two sequential transesterification reactions

#### Small nuclear RNAs in spliceosomes catalyze the splicing of mRNA precursors

#### Transcription and processing of mRNA are coupled

#### Mutations that affect pre-mRNA splicing cause disease

#### Most human pre-mRNAs can be spliced in alternative ways to yield different proteins

### 29.4<SPACE/>The discovery of catalytic RNA was revealing regard to both mechanism and evolution
- 

---

## Lecture notes

### Key points
1. RNA structure: "A" form
2. RNA polymerase
3. promoter mapping
4. stages of transcription
5. termination
6. tRNA structure

### Ribose vs. deoxyribose
- 2' OH vs. 2'-H

### 2<SPACE/>RNA polymerase
- transcription is catalyzed by **RNA polymerase**

$$\ce{(RNA)n + NTP <=> (RNA)_{n+1} + PP_i -> (RNA)_{n+1} + 2 PO4^2-}$$

- comparing DNA polymerase and RNA polymerase
  - in common
    - always 5' to 3' (add to 3')
    - 2 metal centers (2 $\ce{Mg^2+}$)
    - transition state: **pentavalent bipyramidal metaphosphate**
  - differences

|DNA polymerase|RNA polymerase|
|:-------------|:-------------|
|requires primer|does not need primer|
|final product: dsDNA|ssRNA; only copies one strand|
|acts at replication forks|acts at promoters|
|exonuclease 3:5; low error rate|no proofreading; higher error rate|
|copies whole chromosome|stops at termination sites|
|multiple types of DNA polymerases|_E. coli_: only 1 RNA polymerase; _humans_ have 3|

- RNA polymerases
  - **RNA Pol I**. rRNA
  - **RNA Pol II**. mRNA
  - **RNA Pol III**. tRNAs
- **mechanism**
  - metal 1
    - abstract H from 3'-OH
    - orient $\alpha$ phosphate
  - metal 2
    - orient $\alpha, \beta, \gamma$ phosphates
    - stabilize developing negative charge in transition state
- newly formed RNA chains often
  - have triphosphate on the first base
  - the first base is often A or G (pppA or pppG)

#### E. coli (and other bacterial) RNA polymerase
- large: $M_w = 500 kDa$
- polymerase core
  - $\alpha_2$
  - $\beta$
  - $\beta'$
  - $\omega$
- with $\sigma$ factor, called **holoenzyme**
- multiple $\sigma$ subunits
  - $\sigma_{70}$. most common subunit
  - $\sigma_{\text{HS}}$. heat shock

### 3<SPACE/>Promoter mapping
- sequences recognized by $\sigma_{70}$
  - -35: TTGACA
  - -10: TATAAT
- in eukaryotes:
  - -75: CAAT box
  - -10: TATAAA

### 4<SPACE/>Stages of transcription

<Collapse title="Aside: nomenclature">

- **coding/sense/+ strand.**
- **template/antisense/- strand.**
- **transcription unit.** everything from promoter to terminator

</Collapse>

1. **Initiation**: where on DNA does RNA polymerase bind/start?
    - core does **not** recognize promoters
      - binds DNA with $K_d = \pu{5 \times 10^{-12} M}$
    - holoenzyme binds promoters specifically
      - promoter: $\uparrow K_d = \pu{10^{-14} M}$
      - nonpromoter: $\downarrow K_d = \pu{10^{-7} M}$
2. **Elongation**
3. **Termination**

### 5<SPACE/>Termination

### 6<SPACE/>tRNA structure